src/Kalkulator/Kalkulator/Kalkulator.tsx

```diff
@@ -18,8 +24,34 @@ const Kalkulator: FunctionComponent<Props> = ({ restSykefraværshistorikk }) =>
     const [kalkulatorvariant, setKalkulatorvariant] = useState<Kalkulatorvariant>(
         Kalkulatorvariant.Prosent
     );
+    const [sendKalkulatorMetrikker, setSendKalkulatorMetrikker]=useState<boolean>(false);
 
-    useSendIaTjenesteMetrikkMottattVedSidevisningEvent();
```

Comment:
Hvorfor fjernet du hooken? Koden blir jo gjenbrukt også i Kalkulatorrad.

Jeg syntes ikke det er optimalt å ha denne slags businesslogikk definert rett i react-komponentene, som jo er primært ansvarlig for å rendre innhold.